File: app/src/main/java/com/dev/nicola/allweather/weatherProvider/DarkSky/model/Daily.java
package com.dev.nicola.allweather.weatherProvider.DarkSky.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

@SuppressWarnings({"UnusedDeclaration"})
class Daily {

    @SerializedName("summary")
    private String summary;
    @SerializedName("icon")
    private String icon;
    @SerializedName("data")
    private List<DailyData> data = null;

    public Daily() {}

    public String getSummary() {
        return summary;
    }

    public String getIcon() {
        return icon;
    }

    public List<DailyData> getData() {
        return data;
    }
}

File: app/src/main/java/com/dev/nicola/allweather/weatherProvider/DarkSky/model/Daily.kt
package com.dev.nicola.allweather.weatherProvider.DarkSky.model

import com.google.gson.annotations.SerializedName

class Daily {

    @SerializedName("summary")
    val summary: String? = null
    @SerializedName("icon")
    val icon: String? = null
    @SerializedName("data")
    val data: List<DailyData>? = null
}
